lib/os_io.c

```diff
@@ -10487,6 +10658,13 @@ ___processor_state ___ps;)
 {
   ___SCMOBJ e = ___FIX(___NO_ERR);
 
+#ifdef USE_select_or_poll
```

Comment:
actually, this shouldn't change -- we need initialized fdsets even if they are not resizable.